src/update_engine/update_attempter.cc

```diff
@@ -119,9 +117,7 @@ UpdateAttempter::UpdateAttempter(SystemState* system_state,
     status_ = UPDATE_STATUS_UPDATED_NEED_REBOOT;
 }
 
-UpdateAttempter::~UpdateAttempter() {
-  CleanupCpuSharesManagement();
-}
+UpdateAttempter::~UpdateAttempter() {}
```

Comment:
Unless it's the only virtual function override for this class, there's no need for this. When declaring the destructor in .h, use `virtual ~UpdateAttempter() = default`.